### ISSUE TITLE
Improve client validation accessibility on success state in Account Settings page

### ISSUE THREAD

**Issue by ghost** [freeCodeCamp/freeCodeCamp#37377]
**Describe the bug**
Somewhat related to issue #37373. When error state or warning state is triggered by client validation, it is displayed with label color change and an error or warning message. But on success state, there is only label color change without success message.

According to WCAG, **only** changing color is not friendly to color blind or partially color blind users. There should be change in color and shape to accommodate those users.

**To Reproduce**
1. Go to http://localhost:8000/settings
2. Try to trigger success state in `Email Settings` (`Your Internet Presence` client validation is currently not working as described by related issue)
3. Refer to **Describe the bug**

**Expected behavior**
Add success message in addition to color change on success state, or add common semantic icon such as green check icon beside the label.

**Desktop (please complete the following information):**
 - OS: Windows 10 1809 Build 17763.832
 - Browser Latest Chrome
 - Version 77.0.3865.120 (Official Build) (64-bit)

**Comment by rickysyamsi** [comment]
Also calling @ojeytonwilliams for related feedback.

**Comment by ojeytonwilliams** [comment]
Thanks for keeping me in the loop, but I think you've covered all I had in mind and a few things I didn't pick up on!

**Comment by laurameck** [comment]
Taking a look, can I be assigned to this?

**Comment by thecodingaviator** [comment]
@laurameck please pick this up if @ojeytonwilliams hasn't already done so

**Comment by ojeytonwilliams** [comment]
Nope, I haven't had chance.  Don't hesitate on my account.
